backend/main.py

```diff
@@ -23,17 +22,17 @@ class StandardResponse(BaseModel):
 if os.path.exists(".env"):
     load_dotenv(".env")
 
+redis_client = redis.Redis(host="redis", port=6379, db=0)
 
-spotihue = SpotiHue(
+spotihue = spotihue.SpotiHue(
     os.environ.get("SPOTIFY_SCOPE"),
     os.environ.get("SPOTIFY_CLIENT_ID"),
     os.environ.get("SPOTIFY_CLIENT_SECRET"),
     os.environ.get("SPOTIFY_REDIRECT_URI"),
     os.environ.get("HUE_BRIDGE_IP_ADDRESS"),
+    redis_client
```

Comment:
Can we not incorporate it into the class, but pass it in as an optional arg to a method?